src/main/java/edu/kit/ipd/crowdcontrol/objectservice/notification/NotificationPolicy.java

```diff
@@ -0,0 +1,34 @@
+package edu.kit.ipd.crowdcontrol.objectservice.notification;
+
+/**
+ * NotificationPolicy encapsulates strategies to check and send notifications.
+ *
+ * @author Simon Korz
+ * @version 1.0
+ */
+public abstract class NotificationPolicy<T> {
```

Comment:
I didnt want to make it too complicated. And this is kind of a mixture between strategy pattern and template method pattern. Abstract because it assures that invoke does always what it has to do.